### PR TITLE
Remote controls-better ui

### PR DESCRIPTION
**Pull request by makeevrserg**
**Background**

This PR is designed to update update remote-control buttons so it'll be up to date with IRDB-backend

**Changes**

- Add new ButtonData
- Add new icons
- Add new screen with infrared items by long-pressing on brand

**Test plan**

- Open remote controls
- Long press on brands to open all of it's infrareds
- Select infrareds and see new ui

### CODE REVIEW

File: components/keyemulate/impl/src/main/java/com/flipperdevices/keyemulate/helpers/FlipperErrorHelper.kt [hunk 1]
```diff
@@ -31,6 +32,7 @@ class FlipperAppErrorHandlerImpl @Inject constructor() : FlipperAppErrorHelper {
         if (!serviceApi.flipperVersionApi.isSupported(Constants.API_SUPPORTED_FLIPPER_ERROR)) {
             return FlipperAppError.NotSupportedApi
         }
+        Intent.FLAG_FROM_BACKGROUND or Intent.FLAG_GRANT_WRITE_URI_PERMISSION
```
Comment (makeevrserg) [inline components/keyemulate/impl/src/main/java/com/flipperdevices/keyemulate/helpers/FlipperErrorHelper.kt:35]: whoopsie

File: components/remote-controls/api-backend/src/commonMain/kotlin/com/flipperdevices/ifrmvp/api/infrared/model/InfraredHost.kt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 package com.flipperdevices.ifrmvp.api.infrared.model
 
 enum class InfraredHost(val url: String) {
-    DEV("https://infrared.dev.flipp.dev/"),
+    DEV("http://192.168.0.101:8080/"),
```
Comment (makeevrserg) [inline components/remote-controls/api-backend/src/commonMain/kotlin/com/flipperdevices/ifrmvp/api/infrared/model/InfraredHost.kt:4]: whoopsie

File: components/remote-controls/core-model/src/commonMain/kotlin/com/flipperdevices/ifrmvp/model/buttondata/Base64ImageButtonData.kt [hunk 3]
```diff
@@ -7,7 +7,7 @@ import kotlinx.serialization.Serializable
 @Serializable
 data class Base64ImageButtonData(
     @SerialName("key_id")
-    override val keyIdentifier: IfrKeyIdentifier = IfrKeyIdentifier.Unknown,
+    override val keyIdentifier: IfrKeyIdentifier,
```
Comment (makeevrserg) [inline components/remote-controls/core-model/src/commonMain/kotlin/com/flipperdevices/ifrmvp/model/buttondata/Base64ImageButtonData.kt:10]: return back

File: components/remote-controls/core-model/src/commonMain/kotlin/com/flipperdevices/ifrmvp/model/buttondata/ChannelButtonData.kt [hunk 4]
```diff
@@ -8,7 +8,7 @@ import kotlinx.serialization.Serializable
 @Serializable
 data class ChannelButtonData(
     @SerialName("add_key_id")
-    val addKeyIdentifier: IfrKeyIdentifier = IfrKeyIdentifier.Unknown,
+    val addKeyIdentifier: IfrKeyIdentifier,
```
Comment (makeevrserg) [inline components/remote-controls/core-model/src/commonMain/kotlin/com/flipperdevices/ifrmvp/model/buttondata/ChannelButtonData.kt:11]: return back

File: components/remote-controls/core-model/src/commonMain/kotlin/com/flipperdevices/ifrmvp/model/buttondata/OkNavigationButtonData.kt [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class OkNavigationButtonData(
+    @SerialName("up_key_id")
+    val upKeyIdentifier: IfrKeyIdentifier,
+    @SerialName("left_key_id")
+    val leftKeyIdentifier: IfrKeyIdentifier,
+    @SerialName("down_key_id")
+    val downKeyIdentifier: IfrKeyIdentifier,
```
Comment (makeevrserg) [inline components/remote-controls/core-model/src/commonMain/kotlin/com/flipperdevices/ifrmvp/model/buttondata/OkNavigationButtonData.kt:14]: Add empty identifiers as default

File: instances/android/app/src/main/AndroidManifest.xml [hunk 6]
```diff
@@ -3,6 +3,7 @@
         xmlns:tools="http://schemas.android.com/tools">
 
     <application
+            android:usesCleartextTraffic="true"
```
Comment (makeevrserg) [inline instances/android/app/src/main/AndroidManifest.xml:6]: remove

File: components/remote-controls/core-ui/src/main/kotlin/com/flipperdevices/ifrmvp/core/ui/ext/IconTypeExt.kt [hunk 7]
```diff
@@ -27,18 +55,30 @@ fun IconButtonData.IconType.asPainter() = when (this) {
...
-    IconButtonData.IconType.COOL,
-    IconButtonData.IconType.BACK,
-    IconButtonData.IconType.HOME,
-    IconButtonData.IconType.INFO,
-    IconButtonData.IconType.MORE,
-    IconButtonData.IconType.MUTE -> MaterialTheme.colors.onPrimary
-
-    IconButtonData.IconType.POWER -> LocalPalletV2.current.icon.danger.primary
+    IconButtonData.IconType.CAMERA -> rememberVectorPainter(Icons.Default.CameraAlt)
+    IconButtonData.IconType.BRIGHT_MORE -> rememberVectorPainter(Icons.Default.Brightness2)
```
Comment (LionZXY) [inline components/remote-controls/core-ui/src/main/kotlin/com/flipperdevices/ifrmvp/core/ui/ext/IconTypeExt.kt:59]: Reminder: Get icons from the designer

File: components/remote-controls/api-backend/src/commonMain/kotlin/com/flipperdevices/ifrmvp/backend/model/InfraredsResponse.kt [hunk 8]
```diff
@@ -0,0 +1,10 @@
+package com.flipperdevices.ifrmvp.backend.model
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+class InfraredsResponse(
```
Comment (LionZXY) [inline components/remote-controls/api-backend/src/commonMain/kotlin/com/flipperdevices/ifrmvp/backend/model/InfraredsResponse.kt:7]: NIT: Please use `data class`  serializable types

**Review by LionZXY — DISMISSED** [review]

File: components/remote-controls/api-backend/src/commonMain/kotlin/com/flipperdevices/ifrmvp/backend/model/InfraredsResponse.kt (hunk 8 above)
Comment (makeevrserg) [reply components/remote-controls/api-backend/src/commonMain/kotlin/com/flipperdevices/ifrmvp/backend/model/InfraredsResponse.kt]: Fixed

**Review by LionZXY — APPROVED** [review]

**Pull request merged**